### DISCUSSION TITLE
Chrome IOS mobile - lag / laggy transition

### DISCUSSION THREAD

**Discussion by wallythewebdev** [nolimits4web/swiper#5031] [Q&A] [answered]
Hi there: 

On Iphone chrome, Swiper seems to be lagging between transitions of frames: 

I thought this was a V6 issue but I have viewed the Demo pages on Swipers website and the issue is persisting. 

I found this issue that was closed, but a fix had not been provided for chrome on IOS. 
[Swiping on iOS 14.5/14.6 in latest Chrome Version slow and hangs ](https://github.com/nolimits4web/swiper/issues/4493)

Is this an issue that is being addressed in Swiper? 
Does this mean that Swiper is not viable for chrome IOS (as the UX is not great)? 

Thanks - 
W

**Comment by wallythewebdev** [comment, marked as answer]
@nolimits4web - Sorry to @ you on this, but could I please ask for some input, or if you have a found a way around this? 

Edit: 

Reviewed my IOS settings/Version number - 
Updated to the latest version and the issue is resolved. 

Thanks - 
W

**Comment by rossthedevigner** [comment]
@wallythewebdev @nolimits4web I am seeing the same thing only in Chrome iOS v14.5+. Though when I look at other slider/carousel libraries that use translate3d / transition and they all stutter. https://maxmarinich.github.io/react-alice-carousel/
